src/intf/audio/win32/aud_dsound3.cpp

```diff
@@ -3,7 +3,7 @@
 #include "aud_dsp.h"
```

Comment:
Hi Yvan, we've moved the project over to fbneo - you should submit your PR there: 
https://github.com/finalburnneo/FBNeo

Unfortunately, we can't accept this specific one, because we need to retain backwards compatibility with mingw gcc 3.4.5 (2 of the active devs use this one)

best regards,
- dink